bark/models/dynamic/single_track.hpp

```diff
@@ -106,13 +125,19 @@ inline double CalculateSteeringAngle(const SingleTrackModelPtr& model,
   double delta = f_state.angle + atan2(-gain * f_state.lat, vel);
 
   if (limit_steering) {
-    double delta_max =
+    // delta to the left is negative
+    double delta_max_right =
+        std::min(model->GetSteeringAngleMax(),
+                 std::abs(std::atan2(model->GetLatAccelerationRightMax() *
+                                         model->GetWheelBase(),
+                                     vel * vel)));
+    double delta_max_left =
         std::min(model->GetSteeringAngleMax(),
                  std::abs(std::atan2(
-                     model->GetLatAccelerationMax() * model->GetWheelBase(),
+                     model->GetLatAccelerationLeftMax() * model->GetWheelBase(),
```

Comment:
Why Left? -> Side?

Reply:
yes, that's what we get from the RSSResponse